kqueen/auth/common.py

```diff
@@ -9,21 +9,66 @@
 import importlib
 import logging
 
+config = current_config()
 logger = logging.getLogger('kqueen_api')
 
+"""
+Authentication Modules
+
+To define new module, need to specify it as dictionary, where:
+
+"auth_option_lower_case": {"engine": "EqualsToAuthClassName",
+                           "param": {
+                               "key": "value"
+                           }
+                           }
+"""
+
+AUTH_MODULES = {
+    "ldap": {
+        "engine": "LDAPAuth",
+        "param": {
```

Comment:
Maybe use `params` instead of `param` or just full `parameters`. It seems that it was this way before this PR, but while you are refactoring, you could change this.